README.md

```diff
@@ -29,3 +29,10 @@ What is the asymptotic runtime ($\Theta$) of your algorithm in terms of the
 number of comparisons? What is it in terms of the number of flips? Add your
 answer to this markdown file.
 
+## Answer
+
+The runtime I have is $\Theta(n^2)$. Each for loop is making $n$ comparisons. There are two loops, one is nested so $n * n = \Theta(n^2)$. For loops do not always have a time complexity of $\Theta(n)$, and two for loops won't always have a time complexity of $\Theta(n^2)$, but in this situation it does fall under the 'normal case'.
```

Comment:
So the $\Theta(n^2)$ is not the runtime, but the number of comparisons?

Reply:
Yes, that is correct. Sorry if my wording is kind of confusing, is there a better way to say it?

Reply:
Well, if you're analyzing the number of comparison, don't say "runtime"...